python_modules/dagster/dagster/_core/definitions/metadata/types.py

```diff
@@ -0,0 +1,957 @@
+import os
```

Comment:
could this be named `metadata_value.py` for more specificity?

Reply:
yup